client/cypress/integration/dashboard/sharing_spec.js

```diff
@@ -58,6 +58,7 @@ describe("Dashboard Sharing", () => {
       editDashboard();
       cy.getByTestId("AddWidgetButton").click();
       cy.getByTestId("AddWidgetDialog").within(() => {
+        cy.get("input").type("Test Query");
```

Comment:
Is this update related with SQLAlchemy upgrade? If not, please separate another PR.

Reply:
#6693